dev/libsqlite3-pcre/build.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+## This script will ensure that the libsqlite3-pcre dynamic library exists in the root of this
+## repository (either libsqlite3-pcre.dylib for Darwin or libsqlite3-pcre.so for linux). This
+## script is used by run the symbol service locally, which compiles against the shared library.
+##
+## Invocation:
+## - `./libsqlite3-pcre/build.sh`         : build the library
+## - `./libsqlite3-pcre/build.sh libpath` : output its path
+
+cd "$(dirname "${BASH_SOURCE[0]}")/../.."
+set -eu
+
+OUTPUT=`mktemp -d -t sgdockerbuild_XXXXXXX`
+cleanup() {
+    rm -rf "$OUTPUT"
+}
+trap cleanup EXIT
```

Comment:
What you wrote is perfectly fine, but could be made shorter:
```suggestion
trap "rm -rf '$OUTPUT'" EXIT
```
YMMV.